website/blog/2024-01-24-running-tabby-locally-with-rocm/index.md

```diff
@@ -1,10 +1,13 @@
 ---
-slug: running-tabby-locally-with-rocm.md
 title: Running Tabby Locally with AMD ROCm
 authors: [boxbeam]
 tags: [deployment]
 ---
 
+:::warning
+
+Tabby's ROCm support is currently only in our [nightly builds](https://github.com/TabbyML/tabby/releases/tag/nightly). It will become stable in version 0.8.
+
```

Comment:
there's also a closing annotation

<img width="620" alt="image" src="https://github.com/TabbyML/tabby/assets/388154/59f91fd7-37b9-418d-b868-e4fa6f8d7bdd">

Reply:
Thank you, didn't notice that. I'll have to look for a way to preview this next time I'm working on a blog post.